test/jasmine/tests/select_test.js

```diff
@@ -708,4 +722,100 @@ describe('Test select box and lasso per trace:', function() {
         .catch(fail)
         .then(done);
     }, LONG_TIMEOUT_INTERVAL);
+
+    it('should work for bar traces', function(done) {
+        var assertPoints = makeAssertPoints(['curveNumber', 'x', 'y']);
+        var assertRanges = makeAssertRanges();
+        var assertLassoPoints = makeAssertLassoPoints();
+
+        var fig = Lib.extendDeep({}, require('@mocks/0'));
+        fig.layout.dragmode = 'lasso';
+
+        Plotly.plot(gd, fig)
+        .then(function() {
+            return _run(
+                [[350, 200], [400, 200], [400, 250], [350, 250], [350, 200]],
+                function() {
+                    assertPoints([
+                        [0, 4.9, 0.371], [0, 5, 0.368], [0, 5.1, 0.356], [0, 5.2, 0.336],
+                        [0, 5.3, 0.309], [0, 5.4, 0.275], [0, 5.5, 0.235], [0, 5.6, 0.192],
+                        [0, 5.7, 0.145],
+                        [1, 5.1, 0.485], [1, 5.2, 0.40900000000000003], [1, 5.3, 0.327],
```

Comment:
can we use `toBeCloseTo` above in `makeAssertPoints` and avoid `0.40900000000000003` etc?

Reply:
done in 3c73301